packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePoint.java

```
@@ -0,0 +1,82 @@
+  // Used later to always get the correct camera regions instance.
+  private final Callable<CameraRegions> getCameraRegions;
+  private boolean isSupported;
+  private Point currentSetting = new Point(0.0, 0.0);
+
+  public ExposurePoint(
+      CameraProperties cameraProperties, Callable<CameraRegions> getCameraRegions) {
+    this.getCameraRegions = getCameraRegions;
+    this.isSupported = checkIsSupported(cameraProperties);
+  }
+
+  @Override
+  public String getDebugName() {
+    return "ExposurePoint";
+  }
+
+  @Override
+  public Point getValue() {
+    return currentSetting;
+  }
+
+  @Override
+  public void setValue(Point value) {
+    this.currentSetting = value;
+
+    try {
+      if (value.x == null || value.y == null) {
+        getCameraRegions.call().resetAutoExposureMeteringRectangle();
+      } else {
+        getCameraRegions.call().setAutoExposureMeteringRectangleFromPoint(value.x, value.y);
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+  }
+
+  // Whether or not this camera can set the exposure point.
+  @Override
+  public boolean checkIsSupported(CameraProperties cameraProperties) {
+    Integer supportedRegions = cameraProperties.getControlMaxRegionsAutoExposure();
+    final boolean supported = supportedRegions != null && supportedRegions > 0;
+    return supported;
+  }
+
+  @Override
+  public void updateBuilder(CaptureRequest.Builder requestBuilder) {
+    if (!isSupported) {
+      return;
+    }
+
+    Log.i("Camera", "updateExposurePoint | currentSetting: " + currentSetting);
+
+    MeteringRectangle aeRect = null;
+    try {
+      aeRect = getCameraRegions.call().getAEMeteringRectangle();
+      requestBuilder.set(
+          CaptureRequest.CONTROL_AE_REGIONS,
+          aeRect == null
+              ? null
+              : new MeteringRectangle[] {getCameraRegions.call().getAEMeteringRectangle()});
+    } catch (Exception e) {
+      e.printStackTrace();
```

Same here, do we simply ignore the exception and continue executing the capture of the camera?

---

Same as last one here, I think just use dart messenger to send a one time error event back.